twtr/src/integration-test/groovy/twtr/AccountIntegrationSpec.groovy

```diff
@@ -46,19 +46,30 @@ class AccountIntegrationSpec extends Specification {
     }
 
     void "saving an account with a non-unique handle will fail"() {
-        setup:
+        setup: "Add the first account with handle ${handle}"
         def account1 = new Account(handle: handle, emailAddress: email, password: password, displayName: displayName)
         account1.save(flush: true)
+        def startingNumberOfAccount = Account.count()
 
-        when:
+        when: "Another account with ${handle} is attempted to be saved"
         def sus = new Account(handle: handle, emailAddress: email1, password: password1, displayName: displayName1)
+        sus.save(flush: true)
 
-        then:
+        then: "The 2nd account should be invalid and only the 1st account should exist"
         !sus.validate()
+        sus.hasErrors()
+        !sus.id
+        sus.errors.getFieldError('handle')
```

Comment:
One thing I did was verified that the rejectedValue of the field was that of sus.handle to ensure that it rejected the proper value.  sus.errors.getFieldError('handle').rejectedValue == sus.handle